src/pages/Contributing/Designers/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Definition of Done for symbols'
+order:  13
+---
+
+> Mozaic is a worldwide source of truth when it comes to design for Adeo and Leroy Merlin's products. That is why you need to use general guidelines when you contribute to Mozaic. We made some rules to follow when designing new symbols.
+
+This **Definition of Done** page has been divided in 10 steps.
+
+## 1. The pattern doesn't exist in Mozaic
+I checked that the pattern is not in another library :
+- I looked over Mozaic’s website (Foundations / Components)
+- I checked on GitHub (Pull requests)
+- I checked on Jira (Issues in the kanban board)
+
+## 2. The pattern is stored in the right file
+I checked that my pattern is in the right Sketch library and it’s in somewhere that makes sense.
+
+## 3. The naming conventions have been respected in my file
+A design system needs to respond to a certain number of criterias. Naming conventions is one of the most important.
+
+## 4. Every symbols are usable and responsive
+Before asking for review you need to make sure that every symbol in your library is usable and responsive. Usable means that the symbol will be quickly understood by the designers. Responsive means that the symbol can be resized depending on your needs.
+
+## 5. Layer styles and text styles have been applied on the different components
+We created layer and text styles to make the symbol creation process easier. Thank you for making sure to use them 🤘. This will also make the update process way faster.
```

Comment:
maybe you could switch: 
~Thank you for making sure to use them~
for
Make sure to use it, the second time it's usually easier than the first time 🤘. This will...